app/src/main/java/com/example/footprnt/Profile/SavedActivities.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * SavedActivities.java
+ * v1.0
+ * July 2019
+ * Copyright ©2019 Footprnt Inc.
+ */
+package com.example.footprnt.Profile;
+
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.example.footprnt.Discover.Models.Business;
+import com.example.footprnt.Profile.Adapters.SavedActivitiesAdapter;
+import com.example.footprnt.R;
+import com.example.footprnt.Util.AppUtil;
+
+import java.util.ArrayList;
+
+/**
+ * Activity to display the saved activities/events from the discover page
+ *
+ * @author Clarisa Leu
+ */
+public class SavedActivities extends AppCompatActivity {
+
+    private final String TAG = SavedActivities.class.getSimpleName();
```

Comment:
We discussed this briefly earlier but "SavedActivities" is actually more efficient that "SavedActivities.class.getSimpleName()" because .getSimpleName uses reflection